src/main/resources/interface-specification.yml

```
@@ -7796,6 +7886,62 @@ components:
           $ref: '#/components/schemas/AgreementApprovalPolicy'
         eservice:
           $ref: '#/components/schemas/CatalogDescriptorEService'
+    ClientDetails:
```

```suggestion
    Client:
```
After [this](https://github.com/pagopa/interop-be-backend-for-frontend/pull/141#discussion_r1139988911) suggestion has been accepted

---

Client already exists, just spoke with Carmine and FE wants back the entire info linked to Client. However I will check it again with FE guys.